cmd/parca-agent/main.go

```diff
@@ -1032,6 +1036,25 @@ func run(logger log.Logger, reg *prometheus.Registry, flags flags, numCPU int) e
 					default:
 						profilingStatus = profilerStatusInactive
 					}
+					failedReasons := unwindFailedReasons[prflr.Name()][pid]
+					failedReasonsStrs := make([]string, 0)
+					v := reflect.ValueOf(failedReasons)
```

Comment:
I think the idiomatic thing in Go here is to list all fields individually, I realize what you're trying to do here, but I think most people reading this stumble over it.

Reply:
To clarify, are you suggesting I do it like I had in a previous commit (see [here](https://github.com/parca-dev/parca-agent/blob/77aadf96f5086dd6130925e115120c513f9bc7ac/cmd/parca-agent/main.go#L1040)) ? Or are you suggesting something else?

I'm a Go novice, so I'm happy to do whatever you think is the most idiomatic.

Reply:
Yeah I think that would be the typical way to write this in Go.